src/modules/recipe/externalComponentNpm/externalComponentNpm.html

```diff
@@ -0,0 +1,12 @@
+<template>
+    <ui-card title="ExternalComponentNPM">
+        <!-- prettier-ignore -->
+        <div>
+            You loaded this component <relative-time datetime={date} lwc:external></relative-time>
```

Comment:
The behavior of this recipe isn't super clear at first glance if we don't look at the code. The label sticks to "Now" for a minute so the user won't really know that it's dynamic. How about you let them select the date with a `lightning-input` (you can use "now" as the default) and update the text to "Selected date is xxx". This way they will see the text update.